webpack/assets/javascripts/react_app/routes/Audits/AuditsPage/AuditsPage.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { Button, Icon } from 'patternfly-react';
+import { Button } from '@patternfly/react-core';
```

Comment:
This change doesn't seem to be related to the searchbar styling. I get that you want to make the buttons look similar, but that can be done in a different PR.